### PR TITLE
Satisfaction75 Default Keymap Reset

### PR DESCRIPTION
**Pull request by MudkipMao**
## Description

This commit adds a RESET keycode to the default keymap so no one forgets to add it and has to open up their board to manually reset it. Additionally, I changed the encoder resolution from 4 -> 2 because otherwise there are some dead turns on the encoder when the resolution is 4.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/cannonkeys/satisfaction75/config.h [hunk 1]
```diff
@@ -46,6 +46,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* define if matrix has ghost */
 //#define MATRIX_HAS_GHOST
 
+#define ENCODER_RESOLUTION 2
```
Comment (zvecr) [inline keyboards/cannonkeys/satisfaction75/config.h]: Im fairly certain this depends on installed encoder, @awkannan whats your thoughts?
```suggestion
```

Comment (MudkipMao) [reply keyboards/cannonkeys/satisfaction75/config.h]: Oh, ok. I tuned this to the encoder that shipped with the pcb itself: EC11E18244A5

**Comment by noroadsleft** [comment]
#8088 has been merged, and adds a RESET keycode (layer 1 on Left Control).

We could still include the encoder change if it's deemed necessary, but the RESET keycode changes should be removed IMO.

**Comment by zvecr** [comment]
There is also #7758, which covers just the encoder stuff.

**Comment by MudkipMao** [comment]
Cool, looks like everything has been covered. I'll close my PR

**Pull request closed without merging**